megamek/src/megamek/common/actions/compute/ComputeToHit.java

```diff
@@ -794,13 +815,14 @@ public static ToHitData toHitCalc(Game game, int attackerId, Targetable target,
      * @param bArtemisV     flag that indicates whether the attacker is using an Artemis V FCS
      * @param bFTL          flag that indicates whether the attacker is using FTL missiles
      * @param bHeatSeeking  flag that indicates whether the attacker is using Heat Seeking missiles
+     * @param isAradAttack  flag that indicates whether the attacker is using ARAD missiles
      * @param isECMAffected flag that indicates whether the target is inside an ECM bubble
      * @param isINarcGuided flag that indicates whether the target is broadcasting an iNarc beacon
      */
     private static ToHitData compileAmmoToHitMods(Game game, Entity ae, Targetable target, int targetType,
           ToHitData toHit, WeaponType weaponType, Mounted<?> weapon, AmmoType ammoType,
           EnumSet<AmmoType.Munitions> munition, boolean bApollo, boolean bArtemisV, boolean bFTL, boolean bHeatSeeking,
-          boolean isECMAffected, boolean isINarcGuided) {
+          boolean isAradAttack, boolean isECMAffected, boolean isINarcGuided) {
```

Comment:
@HammerGS not for this patch, but we should examine turning all of these booleans into an extensible Record so we can replace the bottom two lines with a single object.

That way we'll be able to add new switches without changing the function signatures.